i18n/en.json

```diff
@@ -1016,7 +1016,7 @@
   },
   {
     "id": "api.command_mute.no_channel.error",
-    "translation": "Could not find the specified channel. Please use the [channel handle](https://about.mattermost.com/default-channel-handle-documentation) to identify channels."
+    "translation": "Could not find the specified channel. Please use the [channel handle](https://docs.mattermost.com/messaging/managing-channels.html#naming-a-channel) to identify channels."
```

Comment:
@cwarnermm @emdecr I noticed while reviewing webapp changes that this redirects to `https://docs.mattermost.com/guides/channels.html#work-with-channels`. Should that be used here instead?